Src/GUI/Plugins/PlotterPlugin.py

```diff
@@ -378,6 +391,7 @@ def __init__(self, parent, title="2D/3D Plotter"):
         wx.Frame.__init__(self, parent, wx.ID_ANY, title, style=wx.DEFAULT_DIALOG_STYLE|wx.MINIMIZE_BOX|wx.MAXIMIZE_BOX|wx.RESIZE_BORDER)
         self.__build_dialog()
         self.__build_menu()
+        self.SetMinSize((-1,800))
```

Comment:
Is there a reason why this has to be added? Adding it means that the size of the Plotter window cannot be reduced vertically at all, which can cause issues on smaller screens (like laptops). Further, it changes the current behaviour of the GUI (which allows resizing in all directions), which could be detrimental/confusing for users who expect/rely on the current behaviour.

Reply:
I see, in that case I would prefer if this were removed. Thanks!